configuration/config-txt/video.md

```diff
@@ -273,10 +238,12 @@ These values are valid if `hdmi_group=1` (CEA):
 | 103 | 2160p | 24Hz | 64:27 | Pi 4 |
 | 104 | 2160p | 25Hz | 64:27 | Pi 4 |
 | 105 | 2160p | 30Hz | 64:27 | Pi 4 |
-| 106 | 2160p | 50Hz | 64:27 | Pi 4 |
-| 107 | 2160p | 60Hz | 64:27 | Pi 4 |
+| 106 | 2160p | 50Hz | 64:27 | Pi 4<sup>2</sup>|
+| 107 | 2160p | 60Hz | 64:27 | Pi 4<sup>2</sup>|
 
-Pixel doubling and quadrupling indicates a higher clock rate, with each pixel repeated two or four times respectively.
+<sup>1</sup>Pixel doubling and quadrupling indicates a pixel clock rate, with each pixel repeated two or four times respectively.
```

Comment:
The text previously said "indicates a higher clock rate" which IMHO makes more sense than "indicates a pixel clock rate".
Did you perhaps mean to write "indicates a higher pixel clock rate" ? :shrug:

Reply:
Yes - fixed.